### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by Rishi-Mishra0704**
- Add styles to the navbar
- Add styles to book section
- Add styles to the form 
- Add a randomizer to change the category
- Add a randomizer to change the percentage
- Add a randomizer to change the chapter number

### CODE REVIEW

File: src/components/css/Book.module.css
Comment (pinkmoon25) [inline src/components/css/Book.module.css:1]: - [ ] Good job styling the application, however as mentioned in the requirements **'Your final design should match Zeplin's design (this includes all elements + the same styling)'** l have some suggestions to make it match the original design please see :point_down: 
image

Tip: you can use [the project created by one of Microverse students - Usman Suleiman Babakolo](https://pedantic-beaver-cb923b.netlify.app/#/). to get the right styling requirements just by inspecting it in the browser.

1. please fix the background color so we see each book card as in the original design
2. The book title seems to be different fro the original design, please fix the font-weight for it
3. Make sure the Add book form matches the original design, including the Input fields and the neccesarry margin and padding(missing space)

Please see the original design :point_down: 
image

**Review by pinkmoon25 — CHANGES_REQUESTED** [review]
Hi @Rishi-Mishra0704 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
Linter checks :heavy_check_mark: 
gitflow :heavy_check_mark: 
functionality :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] Please fix this heading in the Readme so that it mentions the correct project name :point_down: 
image

- [ ] please fix the description here in the readme :point_down: 
image 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/css/Book.module.css
Comment (Rishi-Mishra0704) [reply src/components/css/Book.module.css:1]: @PinkMoon25 I understood the rest of changes but me the input field is looking fine , can you please specify the issue properly?

Comment (pinkmoon25) [reply src/components/css/Book.module.css:1]: hi @Rishi-Mishra0704 , sure the issue is the width please see :point_down: 
image

Comment (Rishi-Mishra0704) [reply src/components/css/Book.module.css:1]: Thank you so much for clarifying

File: src/components/css/Book.module.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
+.book-container {
+  display: flex;
+  margin: 0 10px;
+  margin-bottom: 40px;
+}
```
Comment (nahnahdev) [inline src/components/css/Book.module.css]: - [ ] Great job so far @Rishi-Mishra0704 I see you tried to implement the previous reviewer's suggestion but some of them weren't done properly and in the process, some elements lost their original styles but no need to worry I have made sure to point out each of them in the screenshot below :+1: 
- Your Design
image

- The Original Design
image

**Review by nahnahdev — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @Rishi-Mishra0704 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which haven't been addressed yet.


 ## Highlights
- No linter errors :heavy_check_mark: 
- Good use of GitFlow :heavy_check_mark: 
- Professional README :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @NahnahAJ in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @Rishi-Mishra0704 , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**